source/geod24/LocalRest.d

```diff
@@ -110,30 +110,6 @@ import std.traits : fullyQualifiedName, Parameters, ReturnType;
 import core.thread;
 import core.time;
 
-/// Request / Response ID
```

Comment:
I wish the moving of the symbols was in another commit. It makes it very hard to see what actually changed vs what was only moved.

Reply:
I will keep that in mind.